internal/opa/output.go

```diff
@@ -83,13 +85,22 @@ func prepTemplateFields(a *ast.AnnotationsRef) TemplateFields {
 		// conventions i.e. "package policy.<type>.<name>"
 		shortPath = trimmedPath
 	}
+
+	shortName := getShortName(a)
+
+	var docsUrl string
```

Comment:
It'd be nice to have the URL in Rego in a annotation, perhaps as a followup...

Reply:
Agreed. I added some commentary along those lines.